broker.js

```diff
@@ -260,7 +260,7 @@ class brokerMQTT {
 
             } else {
                 // parse objects to string
-                if (args.mqttMessage && typeof args.mqttMessage !== 'string') {
+                if (args.mqttMessage && typeof args.mqttMessage !== 'string' && typeof args.mqttMessage !== null) {
```

Comment:
From what I know, `typeof SOMETHING` will never evaluate to `null`, so how does this work?

Reply:
should be 'null' (how sloppy), but forget it, re-tested and it's working with the current store version.